test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -439,6 +439,9 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			if newCnfNode == nil {
 				Skip("Skipping the test - cluster does not have another available worker node ")
 			}
+			if profile.Spec.MachineConfigPoolSelector != nil {
```

Comment:
This is fine from code perspective, but are we sure someone will invoke this test without MCPselector set too?

Reply:
yes atleast in our CI we can invoke the test without mcp selector

Reply:
Also the default profile that gets generated by config.go doesn't have mcp selector. So the default profile that gets created when no performance profile exists will have this tests run.